kogito-scripts-springboot/README.md

```diff
@@ -60,3 +71,11 @@ Complete curl command can be found below:
 curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{"name" : "john"}' http://localhost:8080/scripts
 ```
 
+Log after curl executed
+```
+{"id":"ab5239e2-f497-4684-b337-5a44440b38dd","name":"john","message":"Hello john"}
+```
+
+Likewise on Springboot you should see a similar log to
+
+<p align="center"><img src="docs/images/springBootLog.png"></p>
```

Comment:
As for Quarkus and for consitency, would it be possible to put the logs as panel text instead of log screenshot ?

Reply:
fixed @radtriste